goon/browserassets/html/browserOutput.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Chat</title>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<link rel="stylesheet" type="text/css" href="font-awesome.css" />
+	<link rel="stylesheet" type="text/css" href="browserOutput.css" />
+	<script type="text/javascript" src="jquery.min.js"></script>
```

Comment:
Is this another version of jquery, or one of our current 5. These really should be trimmed to one.